sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractSolutionConfiguration.java

```diff
@@ -19,8 +19,7 @@
  */
 package org.sonarsource.dotnet.shared.plugins;
 
-import org.sonar.api.batch.InstantiationStrategy;
-import org.sonar.api.batch.ScannerSide;
+import org.sonar.api.scanner.ScannerSide;
```

Comment:
Ah, I missed existence of this class when adding `AbstractLanguageConfiguration`. Can we remove this class and move it's content there?

I prefer `LanguageConfiguration` over `SolutionConfiguration` in this context.